### DISCUSSION TITLE
Actions without selections

### DISCUSSION THREAD

**Discussion by Synchro** [laravel/nova-issues#5205] [Q&A] [answered]
I have some situations where I want to be able to perform an action (e.g. a global export) without having to select records first. However, the actions menu doesn't appear until I select something. It would be useful to be able to define actions that are not dependent on having a selection so that they can be triggered at any time.

**Comment by crynobone** [comment, marked as answer]
https://nova.laravel.com/docs/4.0/actions/registering-actions.html#standalone-actions

**Reply by Synchro** [reply]
Ah, I'd missed that, perfect, thanks
